internal/server/dispatcher.go

```diff
@@ -68,6 +68,26 @@ func (di Dispatcher) callWebHook(url string, match *match.Transaction) {
 	log.Printf("WebHook response: %d\n", resp.StatusCode)
 }
 
+func (di *Dispatcher) handleCallback(mRequest *mock.Request, mock *mock.Definition) {
+	url := mock.Callback.Url
+	contentType := mock.Callback.ContentType
+
+	di.Evaluator.EvalCallback(mRequest, mock)
+
+	if d := mock.Callback.Delay.Duration; d > 0 {
+		log.Printf("Delaying callback by: %s\n", d)
+		time.Sleep(d)
+	}
+
+	log.Printf("Making callback to %s\n", url)
+	_, err := http.Post(url, contentType, bytes.NewBufferString(mock.Callback.Body))
```

Comment:
It would be nice to allow specify the method, I will be a more generic solution.

Reply:
Yep agreed - as with the other comment, I'm happy to make this PR more flexible. Will update.

Reply:
Updated - now supports any HTTP method